cumulus_library/statistics/psm.sql

```diff
@@ -0,0 +1,52 @@
+        ((cast('Patient/fd32b483921f8c16d0127e9c7f5af3112a973d304fc1ce335b70e439741c8004' AS varchar),cast('Encounter/4cab98a3f5c6ba9439285e19a6ecba6bc50083f613700f992511dc57a572ddcb' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/0b8fed84ab3969013e735852e69d7666c805b11d31200bbab206c3ebf8392e17' AS varchar),cast('Encounter/1e861fb766e92242a3662dc13d885b394d64eeb7ca58a17ceb1129cc49ee9e3f' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/19c65c23167b931cf07f550a8fd3fdb67d634b20e319ce2aa0eab9f22c6ecc3e' AS varchar),cast('Encounter/5e7aa658227c5d4fa139e8382de7a31c31155ca6d8d0a90d5b1580406d23700f' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/de465cd0da7ed926ec5995d7936e825e5510d718624c2cbf9a09432d8c3a1c99' AS varchar),cast('Encounter/a56cd00ceeb6b320ec0f2bb901dd1219e4d909c554e568da4fd5fb8e24b85c1d' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/e3a532c179a12902a2439df569e93e8b48ebed09e84e7a433ff61935f19eff8f' AS varchar),cast('Encounter/7d115ba035c032f49987915075dc007b11253603fa3f898a78013ec66d5bb2a8' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/844fca509614791cc8b9d4fe42dc234599a408bc128d65892a77c550d2c2da8c' AS varchar),cast('Encounter/28a8a7eab0e0585d30b5ce0db87ad022275e44c088e1e301aebe04fab67248e7' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/34cb4e36a97cf468ae7fe1470a66e27613be69a27dbf45e6239b4c3d0573e340' AS varchar),cast('Encounter/633674cb8cc4efe4fd7e9b6ef5985d8c701dd3d37e0b183ba8d2f62aea2ccf3f' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/43e51cdf15d8379e6eae07a0d0e410524dfc2d8546ba214c7c73ecd3ea51011e' AS varchar),cast('Encounter/2fc7a4a9be2f9df9053da6b2f91e63324c8287ee01fff9fb543036a288deca4d' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/3e8dd084158f78dffef04f0a393a8b1ce8710d6345eaeea04d344279076af8f3' AS varchar),cast('Encounter/448f1b4fded6f62e8725dc9f40f6a8a665ba605cdd117b34b0f7fa159f8daf20' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/890668e1eed6daec60fb134725d58b414f4269d829adbf71baf1fe208019955a' AS varchar),cast('Encounter/e643b07cc8ff6979e946ce47b170bae301a4f78fd8bf1b375073edb3ee05b1c7' AS varchar),cast('0' AS varchar))),
+        ((cast('Patient/276e068ce8463e6dd018eca0d2684c928fe69b1a191d966ff3c70b10244310e0' AS varchar),cast('Encounter/515b234f80eb953c2c1ca7629008a9404a5e9b984f7d04e2fbc8c5c89f41193d' AS varchar),cast('0' AS varchar)))
+    )
+        AS t
+        ("subject_ref","encounter_ref","example_diagnosis")
+);
+
+-- ###########################################################
+CREATE TABLE core__psm_encounter_covariate AS (
+    SELECT 
+        DISTINCT sample_cohort."encounter_ref",
+        sample_cohort."example_diagnosis",
+        (
+            SELECT COUNT( DISTINCT encounter_ref )
+            FROM "core__condition"
+            WHERE sample_cohort."subject_ref" = "core__condition"."subject_ref"
+            --AND sample_cohort.enc_end_date >= "core__condition".recordeddate
```

Comment:
Is this (and the other line below) manually commented out? :scissors:?

Reply:
got those, probably while you were looking at this.